www/js/goals.js

```diff
@@ -129,14 +141,118 @@ angular.module('emission.main.goals',['emission.services', 'emission.plugin.logg
     /*$scope.data = {
         showDelete: false
     };*/
+    $scope.toBrowser = function() {
+        var settings = {
+                auth: {
+                    apiId: "7ce7f19a-4668-4ba3-85f8-dd588910b9d0",
+                    apiToken: "f3964931-174b-4366-96e9-1e2a7896d6e0"
+                }
+            };
+            var options = {
+              location: 'yes',
+              clearcache: 'yes',
+              toolbar: 'yes'
+            };
 
+
+
+          $rootScope.$on('$cordovaInAppBrowser:loadstart', function(e, event){
+            // insert Javascript via code / file
+            $cordovaInAppBrowser.executeScript({
```

Comment:
Also, where is the callback for the `executeScript`? The raw IAB has a callback function. You are using the angular wrapper, so presumably it returns a promise. Where's the `then` for the promise?